doc/contributing/documentation_guide.md

```diff
@@ -217,57 +217,83 @@ For methods written in C, \RDoc cannot determine what arguments
 the method accepts, so those need to be documented using \RDoc directive
 [`:call-seq:`](rdoc-ref:RDoc::Markup@Method+arguments).
 
+For a singleton method, use the form:
+
+```c
```

Comment:
These code blocks are not valid C, we should just leave them as plain text.

Reply:
Fixed (all instances).